Policies/github-corporate-terms-of-service.md

```diff
@@ -25,23 +25,29 @@ Thank you for choosing GitHub for your company's business needs. Please read our
+5. "Customer", “You,” and “Your” refer to the company, or organization that has visited or is using the Website or Service; that accesses or uses any part of the Account; or that directs the use of the account in the performance of its functions. Special terms may apply for business or government accounts (See [Section B(7): Additional Terms](#7-additional-terms)).
+6. "User" refers to the individual who has visited or is using the Website or Service on your behalf; that accesses or uses any part of the account on your behalf; or that directs the use of the account in the performance of their functions on your behalf. All Users must be at least 13 years of age. "Other Users" refers to the individuals, not including your Users, who visit or use the Website or Service. “Users” and “Other Users” are collectively referred to as “All Users”.
+7. “GitHub,” “We,” and “Us” refer to GitHub, Inc., as well as our affiliates, directors, subsidiaries, contractors, licensors, officers, agents, and employees.
+8. An "Account" represents your legal relationship with GitHub. A “User Account” represents an individual User’s authorization to log in to and use the Service and serves as a User’s identity on GitHub. A "Corporate Account" refers to an Account created by a User on behalf of an entity, such as a company, non-profit organization, or group. A Corporate Account may include an Enterprise Cloud subscription or a Team plan. “Organizations” are shared workspaces that may be associated with a single entity or with one or more Users where multiple Users can collaborate across many projects at once. A User Account can be a member of any number of Organizations.
+9. “Content” refers to content featured or displayed through the Website, including without limitation text, data, articles, images, photographs, graphics, software, applications, designs, features, and other materials that are available on the Website or otherwise available through the Service. “User-Generated Content” is Content, written or otherwise, created or uploaded by All Users. "Your Content" is Content that you create, own, or to which you are the rights holder.
+10. "Professional Services" means training, consulting, or implementation services that GitHub provides to you pursuant to a mutually executed SOW. Professional Services do not include Support.
+11. "SOW" means a mutually executed statement of work detailing the Professional Services GitHub will perform for you, any related Fees, and each party's related obligations.
```

Comment:
If you change all these list items to be `1.`, future additions won’t require changing the whole list:

```markdown
1. foo
1. bar
1. baz
```

becomes:

> 1. foo
> 1. bar
> 1. baz

Reply:
I'm pretty sure that applies only to comments, but not to the contents of the files being diffed, which do not get processed further.

Reply:
It would be nice to get a diff word-by-word so we can see just what has changed, though, without reading paragraphs at a time.

Reply:
Looks like it’s rendered to me: https://help.github.com/en/articles/github-corporate-terms-of-service

Reply:
@j-f1 Thanks for the recommendation and feedback! We won't be making the formatting change at this time, but we'll take a look over the coming week, and may add it in prior to merging this PR next Friday if the formatting works out.